workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+``` 
+All good ? 
+
+# Now, lets move on to Next file : ChristmasTree.py
+
+
+Now open the main.py 
+
+In this page we are gonna make a tree stem made of rectangle then we are gonna make multiple rectangles stacked above each other to make it look like a christmas tree. 
+Shall we begin ?
+
+```python
+from turtle import *
+from Figures import *
+from random import randint
+
+```
+Here, we import all fuctions from 'turtle', all functions from 'Figures' and finally 'randint' function from 'random' so that we can use it to generate a random integer.
+Why do we need a random integer ? We'll see .
+
+
+```python
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+```
+******speed()****** function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
+Next we set the background color to maroon using th hexcode.
+Now decide a value for ******y******, which is basically trial and error. Set some random value first and then iterate to find the right vaue which brings your turtle to the center.
+In Case of ******width******, provide any value you desire. 
+
+```python
+# Constructing the tree
+box1(turtle, "#4F4553", -15, y - 40, 30, 40) #Stem of the tree
+```
+Now we make a make a stem for the tree by passing desired values for each parameter. Color will be a shade of brown and the rest as you see above. 
+![alt text](https://cloud-4ipo9natc.vercel.app/4image3.png) 
+
+
+Next thing we are gonna make is the tree leaves with balls on either sides.
+This is the only part you have to be a bit careful while doing. See if the code is undertandable.
+
+```python
+while width > 20 :
+    width = width - 20
+    height = randint(20,30)
+    x = 0 - width/2
+    box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+    balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+    balls(turtle, "#fc045b", -x, y, 5) # Decorative balls
+    y = y + height
+
+```
+Not too difficult, is it ? Let's break this down.
+
+
+```python
+width = width - 20
+height = randint(20,30)
+```
+In each iteration, the width of the next rectangle gets reduced to a smaller size so that in the end we get a triangle kinda look for the entire leaf part (green portion). We assign a random value for the height as that helps to give the tree an irregular shape rather than a giving same height for all the rectangles. If you would like, you could reduce the width by a random integer too. 
+
```

Comment:
```suggestion
In each iteration, the width of the next rectangle gets reduced to a smaller size so that in the end we get a triangle kinda look for the entire leaf part (green portion). We assign a random value for the height as that helps to give the tree an irregular shape rather than giving the same height for all the rectangles. If you would like, you could reduce the width by a random integer too. 

```